TSC-Charter.md

```diff
@@ -56,14 +57,25 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC may, at its discretion, invite any number of non-voting observers to
+participate in the public portion of TSC discussions and meetings.
 
 The TSC shall meet regularly using tools that enable participation by the
 community (e.g. weekly on a Google Hangout On Air, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
-the TSC Chairperson. Minutes or an appropriate recording shall be taken
-and made available to the community through accessible public postings.
+the TSC Chairperson. Responsibility for directing individual meetings may be
+delegated by the TSC Chairperson to any other TSC member. Minutes or an
+appropriate recording shall be taken and made available to the community
+through accessible public postings.
+
+In the case where an individual TSC member attends fewer than 25% of the
+regularly scheduled meetings held within a six month period, does not attend
+at least one meeting every three months, and does not proactively communicate
+with the TSC their intention not to attend those meetings, the member shall be
+asked to voluntarily resign their membership. Should a member fail to attend
+100% of the meetings within a six month period, the member shall be
```

Comment:
We should change this to "any" meeting. Right now it reads like you must attend *every* meeting or be removed, which contradicts the line above and is certainly not our expectation.